test-e2e/memory/index.ts

```diff
@@ -1 +1,8 @@
-export default class Memory {}
+import fs from 'fs/promises';
+
+export default class Memory {
+  json = async (path: string): Promise<any> => {
```

Comment:
we can avoid using `async` here if we replace `fs.readFile(path, 'utf-8');` by `fs.readFileSync(path, 'utf-8');`

Reply:
I dont want to block IO, anyway computed supports async methods

Reply:
just curious do you mean that non blocking IO operations will improve something in that function? =)

Reply:
no, I don't think so. Pretty much sure that we never have cases when sync method will be slower than async.

P.S this json function is only for testing purposes, later it will be added to cli module

Reply:
completely agree with you, for that function there will be no difference in blocking vs non-blocking IO, so thanks for conversation, will close that comment =)